### PR TITLE
DOCS-10403 - Updates to UUID method

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
Updating UUID method input params and providing additional examples.

Staging: https://docs-mongodbcom-staging.corp.mongodb.com/jeffreyallen/DOCS-10403/reference/method/UUID.html

Code review: https://mongodbcr.appspot.com/158930001/

---
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/2973)

### CODE REVIEW

**Comment by jeff-allen-mongo** [comment]
Hey @kay-kim this is ready for a final look and merge

File: source/includes/apiargs-method-UUID-param.yaml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 arg_name: param
 description: |-
-  Specify a 32-byte hexadecimal string to convert to the UUID BSON subtype.
+  Specify a 36 character string to convert to a UUID BSON object. If
+  not provided, MongoDB generates a random UUID in 
+  `RFC 4122 v4 <https://tools.ietf.org/html/rfc4122>`_. format.
```
Comment (kay-kim) [inline source/includes/apiargs-method-UUID-param.yaml]: extra period before format

Comment (jeff-allen-mongo) [reply source/includes/apiargs-method-UUID-param.yaml]: oops

**Comment by jeff-allen-mongo** [comment]
@kay-kim, ready for another look based on your feedback 🙂

**Pull request merged**